app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command) != 3:
```

Comment:
The wrong check!
command is a string
It works incorrectly. Did you test your code manually? 
For example "mv file1.txt file2.txt"